### PR TITLE
Adding ical-Output

### PR DESCRIPTION
**Pull request by migmedia**
Hi,

I have needed the ability to output some ics-files. Maybe it's helpfull to others?

migmedia

### CODE REVIEW

**Comment by Peltoche** [comment]
Hello everyone, sorry for the delay.

I didn't have much time for open-source lately. I will try to check this today and I hope I will be able to release it too.

**Review by Peltoche — APPROVED** [review]

**Comment by Peltoche** [comment]
Hi,

Everything looks good to me. A big thanks @migmedia for this contribution :+1: .

Could you rebase this branche please? Once this is done I will merge it and create a new release.

File: Cargo.toml [hunk 1]
```diff
@@ -17,18 +17,16 @@ version = "0.6.0"
...
 
-
-
 [dependencies]
 thiserror = "1.0.20"
 
-
 [features]
-default = ["vcard", "ical"]
+default = ["ical", "vcard"]
```
[the remark this answers is not in the document]
Comment (migmedia) [reply Cargo.toml:25]: If @Peltoche thinks it is a default, he can change that. It's his project and I didn't want to force it in a new directtion.

**Comment by migmedia** [comment]
> Could you rebase this branche please? Once this is done I will merge it and create a new release.

Due to the merge (370d9db ) caused by a conflict in `Cargo.toml` rebasing would change history and destroy the clones of this repository. Or do I misunderstand you?

**Comment by Peltoche** [comment]
You need to rebase your master on top mine and resolve the conflict between the two in order to keep your new commits on top of my HEAD and be able to push them. This will make some changes in your last commits on your master. I you don't want that you can make a new branche on your side and make the changes on this branche. Once you fork a repo you need to keep up with the new reviews if you want to be able to merge.

**Comment by migmedia** [comment]
Created a new branch on master/HEAD and started a new [pull-request](/Peltoche/ical-rs/pull/49).

**Pull request closed without merging**